src/components/CalendarMonthGrid.jsx

```diff
@@ -166,6 +189,20 @@ export default class CalendarMonthGrid extends React.Component {
     this.container = ref;
   }
 
+  setMonthHeight(height, i) {
+    if (this.calendarMonthHeights[i]) {
+      if (i === 0) {
+        this.calendarMonthHeights.pop();
+        this.calendarMonthHeights.unshift(height);
```

Comment:
the mutation, it makes me sad. is there no way to avoid or simplify all this?

Reply:
I can make a new array instead.